doc/source/myrocks/variables.rst

```diff
@@ -0,0 +1,2255 @@
+.. _myrocks_variables:
+
+=================
+MyRocks Variables
+=================
+
+MyRocks provides variables to tune performance
+and manage the behavior of the storage engine.
+
+.. contents::
+   :local:
+
+.. _myrocks_server_variables:
+
+MyRocks Server Variables
+------------------------
+
+The MyRocks server variables expose configuration
+of the underlying RocksDB engine.
+There several ways to set these variables:
+
+* For production deployments,
+  you should have all variables defined in the configuration file.
+
+* *Dynamic* variables can be changed at runtime using the ``SET`` statement.
+
+* If you want to test things out, you can set some of the variables
+  when starting ``mysqld`` using corresponding command-line options.
+
+  .. note:: Not all variables have a corresponding option
+     that can be set on the command line.
+
+If a variable was not set in either the configuration file
+or as a command-line option,
+the default value is used.
+
+Also, all variables can exist in one or both of the following scopes:
+
+* *Global* scope defines how the variable affects overall server operation.

```

Comment:
Is the MySQL variable 101 discussion needed there?

Reply:
It's very simple, adds some clarity IMHO